include/deal.II/numerics/vector_tools_integrate_difference.templates.h

```diff
@@ -461,8 +460,10 @@ namespace VectorTools
                  ExcDimensionMismatch(weight->n_components, n_components));
         }
 
+      // Reinit the difference vector
       difference.reinit(dof.get_triangulation().n_active_cells());
 
+      // Change the exponent if necessary
```

Comment:
The comments are correct but they aren't necessary - they state almost exactly what happens in the next line. I would prefer not to add them since there is very little benefit to the reader (e.g., `difference.reinit()` is the same as `"reinit the difference vector").

The same goes for `Collect the UpdateFlags for the norm` below.

Reply:
Comments are dropped.